### PR TITLE
aboutContent and faq components for better user experience

### PR DESCRIPTION
**Pull request by KD63799**
This pull request introduces several enhancements to the AboutContent and FAQ components of our project, improving readability and user interaction. Key updates include:

**AboutContent Component Enhancements:**

Refined the narrative in the history section, enhancing the storytelling aspect and providing a more engaging user experience.
Updated team member display, ensuring each member's information is presented clearly and concisely.
Improved responsive design elements for better usability across various devices.

**FAQ Component Updates:**

Implemented useState and AnimatePresence from Framer Motion to handle FAQ interactions, providing a smoother and more interactive accordion effect.
Organized FAQ content to ensure answers are accessible and user-friendly.
Added dynamic rendering of FAQs to enable easy updates and management of content.
These enhancements are designed to make the application more intuitive and enjoyable for users, encouraging longer engagement and repeat visits.

### CODE REVIEW

File: frontend/src/components/Footer/Footer.jsx
Comment (Childbutsmart) [inline frontend/src/components/Footer/Footer.jsx:1]: The links should be in kebab case

File: frontend/src/components/NavBar/NavBar.jsx
Comment (Childbutsmart) [inline frontend/src/components/NavBar/NavBar.jsx:1]: The links should be in kebab case

**Review by Childbutsmart — CHANGES_REQUESTED** [review]
The links in router.js should be in kebab case.

**Review by CalliEtc — APPROVED** [review]

**Review by Childbutsmart — APPROVED** [review]
Great job mate

**Review by jlaron230 — APPROVED** [review]

**Pull request merged**